app/views/positive_charge_adjust_forms/new.html.erb

```diff
@@ -1,61 +1,70 @@
         <% end %>
 
-        <%= f.label :amount, class: "form-label" do %>
+        <%= f.label :amount, class: "govuk-label" do %>
           <%= t(".fields.amount.label") %>
-          <span class="form-hint"><%= t(".fields.amount.hint") %></span>
+          <span class="govuk-hint"><%= t(".fields.amount.hint") %></span>
         <% end %>
-        <%= f.number_field :amount, class: "form-control", step: "0.01" %>
+        <%= f.number_field :amount, class: "govuk-input govuk-input--width-10", step: "0.01" %>
       </div>
 
-      <div class="form-group <%= "form-group-error" if @positive_charge_adjust_form.errors[:reference].any? %>" id="reference">
+      <div class="govuk-form-group <%= "govuk-form-group--error" if @positive_charge_adjust_form.errors[:reference].any? %>" id="reference">
         <% if @positive_charge_adjust_form.errors[:reference].any? %>
-          <span class="error-message"><%= @positive_charge_adjust_form.errors[:reference].join(", ") %></span>
+          <span class="govuk-error-message"><%= @positive_charge_adjust_form.errors[:reference].join(", ") %></span>
         <% end %>
 
-        <%= f.label :reference, t(".fields.reference.label"), class: "form-label" %>
-        <%= f.text_field :reference, class: "form-control" %>
+        <%= f.label :reference, t(".fields.reference.label"), class: "govuk-label" %>
+        <%= f.text_field :reference, class: "govuk-input govuk-input--width-10" %>
       </div>
 
-      <div class="form-group <%= "form-group-error" if @positive_charge_adjust_form.errors[:description].any? %>" id="description">
-        <% if @positive_charge_adjust_form.errors[:description].any? %>
-          <span class="error-message"><%= @positive_charge_adjust_form.errors[:description].join(", ") %></span>
-        <% end %>
+      <div class="govuk-character-count"  data-module="govuk-character-count" data-maxlength="500">
+        <div class="govuk-form-group <%= "govuk-form-group--error" if @positive_charge_adjust_form.errors[:description].any? %>" id="description">
+          <% if @positive_charge_adjust_form.errors[:description].any? %>
+            <span class="govuk-error-message"><%= @positive_charge_adjust_form.errors[:description].join(", ") %></span>
+          <% end %>
 
-        <%= f.label :description, class: "form-label" do %>
-          <%= t(".fields.description.label") %>
+          <%= f.label :description, class: "govuk-label" do %>
+            <%= t(".fields.description.label") %>
 
-          <span class="form-hint">
-            <%= t(".fields.description.hint") %>
-          </span>
-        <% end %>
+            <span class="govuk-hint">
+              <%= t(".fields.description.hint") %>
+            </span>
+
+            <div id="with-hint-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
+              You can enter up to 200 characters
```

Comment:
Should this text be in an I18n file instead?

Reply:
That text shouldn't be there. Well spotted!. The <div> is there to help the 'You have X characters remaining' text.